metadata-ingestion/src/datahub/ingestion/source/redash.py

```diff
@@ -725,8 +742,12 @@ def _emit_chart_mces(self) -> Iterable[MetadataWorkUnit]:
 
                     yield wu
 
+    def add_config_to_report(self) -> None:
```

Comment:
Why is this a different method?

Reply:
We used this pattern in `snowflake` and `bigquery` connector. In those we used it separately so did the same. This is basically to show that we are just copy-pasting some stuff